examples/test.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+include '../vendor/autoload.php';
+
+$api = new \atk4\api\Api();
+
+class Country extends \atk4\data\Model
```

Comment:
Probably test model should be moved somewhere outside test script.

Reply:
This was a first quick test. Probably would remove this file long-term or clean it up.